<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1001</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <widget class="QTableWidget" name="tbl_dept_info"/>
    </item>
    <item>
     <widget class="QTableWidget" name="tbl_dept_sum_info">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_inform">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>막대그래프를 클릭 하면 해당월의 파이차트를 확인 할 수 있습니다.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>999</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="chart_layout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
